### DISCUSSION TITLE
Changing material properties

### DISCUSSION THREAD

**Discussion by Nikhil23121998** [peridigm/peridigm#269] [General]
If i have to change elastic modulus in peridigm/examples/tensile_test sample, which file i have to modify?

**Comment by oldninja** [comment]
The yaml input file.

**Reply by oldninja** [reply]
Line 34
https://github.com/peridigm/peridigm/blob/master/examples/tensile_test/tensile_test.yaml#L34

**Reply by Nikhil23121998** [reply]
Thank you

**Comment by Nikhil23121998** [comment]
I am trying to change bulk modulus and shear modulus values in YAML code, but it is giving same results.

**Reply by ppandit95** [reply]
because this values are calculated inside the pridigm code once you specify young's modulus and poisson's ratio for an isotropic material but instead you can just specify bulk and shear modulus in place of young's modulus and poison's ratio

Hope this helps.

Regards
Pushkar
